Description: Upstream changes introduced in version 2.07-3
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 fbi (2.07-3) unstable; urgency=low
 .
   * Switch to source format 3.0 (quilt)
 .
 The person named in the Author field signed this changelog entry.
Author: Moritz Muehlenhoff <jmm@debian.org>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- /dev/null
+++ fbi-2.07/Make.config
@@ -0,0 +1,16 @@
+LIB		:= lib
+RESDIR		:= /etc/X11
+HAVE_ENDIAN_H	:= yes
+HAVE_LINUX_FB_H	:= yes
+HAVE_NEW_EXIF	:= yes
+HAVE_GLIBC	:= yes
+HAVE_STRCASESTR	:= yes
+HAVE_LIBPCD	:= yes
+HAVE_LIBUNGIF	:= yes
+HAVE_LIBPNG	:= yes
+HAVE_LIBTIFF	:= yes
+#HAVE_LIBMAGICK	:= no
+HAVE_LIBSANE	:= no
+HAVE_LIBCURL	:= yes
+HAVE_LIBLIRC	:= no
+HAVE_MOTIF	:= no
